assets/script.js

```diff
@@ -45,7 +45,7 @@ const getArchiveCards = i => {
 
   document.getElementById(`archiveObject_${i}`).onload = function() {
     const archiveObject = document.getElementById(`archiveObject_${i}`)
-    const cards = archiveObject.contentDocument.querySelectorAll('.card')
+    const cards = archiveObject.querySelectorAll('.card')
```

Comment:
Hey this change is definitely not needed. It causes a problem. The `contentDocument` is needed because we insert the cards into an `<object>` element which indeed behaves like an iframe, and we need contentDocument to access its children. 
Thanks for attempting to improve the code but in this case we have to skip this change. The card contribution on the other hand is great. If you can remove this change and commit again the PR should update and leave us only with the card and then we can merge it

Reply:
Hey this change is definitely not needed. It causes a problem. The contentDocument is needed because we insert the cards into an <object> element which indeed behaves like an iframe, and we need contentDocument to access its children.
Thanks for attempting to improve the code but in this case we have to skip this change. The card contribution on the other hand is great. If you can remove this change and commit again the PR should update and leave us only with the card and then we can merge it